easybuild/easyconfigs/r/RCall/RCall-0.13.17-foss-2022a-R-4.2.1-Julia-1.9.2.eb

```diff
@@ -224,17 +222,13 @@ exts_list = [
     }),
 ]
 
-sanity_check_commands = ["julia -e 'using Pkg;Pkg.test(\"%(name)s\")'"]
+sanity_check_commands = [
+    """julia -e 'using Pkg; Pkg.activate("%(installdir)s/environments/v1.9"); Pkg.test("%(name)s")'""",
```

Comment:
can we avoid hardcoding v1.9 here? e.g.
```python
_julia_shortver = '.'.join(_julia_ver.split('.')[:2])
```

Reply:
good idea, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/20106/commits/5a5f662d3272c63b5523bb87e8c966554e15906e